tools/picard/picard_macros.xml

```diff
@@ -16,6 +16,10 @@
         </requirements>
     </xml>
 
+    <token name="@symlink_element_identifier@"><![CDATA[
+    ln -f -s '$inputFile' '$inputFile.element_identifier' &&
+    ]]></token>
```

Comment:
This must be escaped in my opinion https://github.com/galaxyproject/tools-iuc/pull/1019.